src/components/dialogs/parameters/common/provider-parameter.tsx

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright (c) 2025, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+import { Grid, SelectChangeEvent } from '@mui/material';
+import { DropDown } from '../parameters';
+import { useCallback } from 'react';
+
+export interface ProviderParameterProps {
+    providers: Record<string, string>;
+    provider: string | undefined;
+    onChangeProvider: (provider: string) => void;
+}
+
+export default function ProviderParameter({ providers, provider, onChangeProvider }: Readonly<ProviderParameterProps>) {
```

Comment:
Why do you have a different compenent for dynamic providers ?

Reply:
providers in others calculations are recently migrated to rhf. I will profit this PR to migrate ones for dynamic calculations

DONE!  in https://github.com/gridsuite/gridstudy-app/pull/2499/commits/a66f3ed73c07a1b6548fccbfcb1f8608197bf368